app/code/community/Mbiz/Reports/Model/Sales/Journal/Result/Payment/Method.php

```diff
@@ -36,7 +36,7 @@ protected function _load(\Mbiz_Reports_Model_Sales_Journal_Request $request, \Ze
                 'discount' => 'SUM(ROUND(invoice.base_discount_amount, 2))',
                 'shipping' => 'SUM(ROUND(invoice.shipping_amount, 2))',
                 'tax'      => 'SUM(ROUND(invoice.base_tax_amount, 2))',
-                'total'    => 'SUM(ROUND(invoice.grand_total, 2))',
+                'total'    => 'SUM(ROUND(invoice.base_grand_total, 2))',
```

Comment:
Can you add the `IFNULL` statement to all these sums please?

I know it wasn't there, but we can improve :)